src/Frontend/src/router/routeLinks.ts

```diff
@@ -33,7 +33,8 @@ const failedMessagesLinks = (root: string) => {
 const messagesLinks = (root: string) => {
   return {
     root,
-    message: { link: (id: string) => `${root}/${id}`, template: "/messages/:id" },
```

Comment:
The way success messages vs failed messages are loaded is different hence the 2 different options